src/ui/components/common/creator_listing.rs

```diff
@@ -0,0 +1,33 @@
+use crate::utils::create_user_apub_name;
+use leptos::*;
+use leptos_router::A;
+
+#[component]
+pub fn CreatorListing(
+  avatar: Option<String>,
+  display_name: Option<String>,
+  name: String,
+  actor_id: String,
```

Comment:
You can probably pass the whole person. 

[Example from jerboa](https://github.com/LemmyNet/jerboa/blob/main/app/src/main/java/com/jerboa/ui/components/person/PersonProfileLink.kt#L66)

Reply:
[Similar issue to here.](https://github.com/LemmyNet/lemmy-ui-leptos/pull/107#discussion_r1694356214)